types/storybook__vue/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for @storybook/vue 3.0
+// Project: https://github.com/storybooks/storybook
+// Definitions by: Yutaro Miyazaki <https://github.com/vwxyutarooo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+/// <reference types="webpack-env" />
```

Comment:
Does this package really have anything to do with webpack? What does `configure` require its second argument to have?

Reply:
Yeah, config is actually usig outside of `storybook__vue-test.ts` which look like [this](https://github.com/storybooks/storybook/blob/master/examples/vue-kitchen-sink/.storybook/config.js). Source code is [here](https://github.com/storybooks/storybook/blob/b4dec5fa488727391ab7a7fb1767cdf10f5093ad/lib/core/src/client/preview/config_api.js#L34).